core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/Integration.kt

```diff
@@ -203,6 +209,19 @@ internal class Integration(
             }
         }
 
+        notebook.renderersProcessor.registerWithoutOptimizing(
```

Comment:
It where a way to override context renderer only in JupyterHtmlRenderer? 
As i see it users will start to see images resized to 100 px if they use dataframe even outside HTML table

Reply:
I fixed that